README.md

```diff
@@ -122,7 +121,7 @@ If you need to change all of history, see the man page for 'git filter-branch'.
 In order to remove a file from a commit, do the following:
 
 ```sh
-$ git checkout HEAD~2 myfile
+$ git checkout HEAD^ myfile
```

Comment:
I also believe I fixed a bug. This checkout was supposed to reference a *previous* commit, i.e. `HEAD^` or `HEAD~1`, right?